api/src/data/migrations/020.create-role-logs.ts

```diff
@@ -0,0 +1,40 @@
+import * as knex from "knex";
+
+export async function up(knex: knex.Knex) {
+    await knex.schema.createTable("role_logs", function(table) {
```

Comment:
@datajohnson Should we change this to a json column `changes`? See [PaperTrail](https://github.com/paper-trail-gem/paper_trail?tab=readme-ov-file#metadata-can-override-papertrail-columns) ruby gem for an example.

Reply:
I argued to store these logs tables differently and lost. They will unfortunately stay as designed by Peter.

Reply:
It's a bit ironic; sometimes I've advocated for storing logs this way, but never actually seen how messy it looks all written out. But it will make any future log compatibility very straight forward so that's good at least.